sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/AvroGenericRecordToStorageApiProto.java

```diff
@@ -350,10 +350,11 @@ private static TableFieldSchema fieldDescriptorFromAvroField(org.apache.avro.Sch
           throw new RuntimeException("Unexpected null element type!");
         }
         TableFieldSchema keyFieldSchema =
-            fieldDescriptorFromAvroField(new Schema.Field("key", keyType, "key of the map entry"));
+            fieldDescriptorFromAvroField(
+                new Schema.Field("key", keyType, "key of the map entry", null));
```

Comment:
a `null` value is what Avro 1.11 defaults to, so there's no behavior change here: https://github.com/apache/avro/blob/release-1.11.4/lang/java/avro/src/main/java/org/apache/avro/Schema.java#L588-L589